src/Compilers/Core/Portable/WellKnownMembers.cs

```diff
@@ -2968,14 +2968,29 @@ static WellKnownMembers()
                  0,                                                                                                                                             // Arity
                     0,                                                                                                                                          // Method Signature
                     (byte)SignatureTypeCode.TypeHandle, (byte)SpecialType.System_Int32,
-                    
-                 // System_Runtime_CompilerServices_IsUnmanagedAttribute__ctor
+
+                // System_Runtime_CompilerServices_IsUnmanagedAttribute__ctor
                  (byte)(MemberFlags.Constructor),                                                                                                               // Flags
                  (byte)WellKnownType.ExtSentinel, (byte)(WellKnownType.System_Runtime_CompilerServices_IsUnmanagedAttribute - WellKnownType.ExtSentinel),       // DeclaringTypeId
                  0,                                                                                                                                             // Arity
                      0,                                                                                                                                         // Method Signature
                      (byte)SignatureTypeCode.TypeHandle, (byte)SpecialType.System_Void,
 
+                 // Microsoft.VisualBasic.Conversion.Fix(Number As System.Single)
+                (byte)(MemberFlags.Method | MemberFlags.Static),                                                                                    // Flags
+                 (byte)WellKnownType.ExtSentinel, (byte)(WellKnownType.Microsoft_VisualBasic_Conversion - WellKnownType.ExtSentinel),               // DeclaringTypeId
+                0,                                                                                                                                  // Arity
+                    1,                                                                                                                              // Method Signature
+                    (byte)SignatureTypeCode.TypeHandle, (byte)SpecialType.System_Single,                                    // Return type
```

Comment:
>// Return type [](start = 124, length = 14)

Please align the comment with a comment on the previous line

Reply:
This indentation was intentional, and corresponds to the indentation on the left.

Reply:
Look at the alignment on the left. The method signature part is indented differently. The same is true here for the comment on the right.

---
In reply to: [188366864](https://github.com/dotnet/roslyn/pull/26846#discussion_r188366864) [](ancestors = 188366864,188352605)

Reply:
The indentation here is intentional. The data for the method signature are not indented the same on the left, and the comments for the method signature, forming a separate self-contained data block, are indented differently on the right.

---
In reply to: [188368103](https://github.com/dotnet/roslyn/pull/26846#discussion_r188368103) [](ancestors = 188368103,188366864,188352605)

Reply:
> Look at the alignment on the left. The method signature part is indented differently. The same is true here for the comment on the right.

Please follow the established pattern in this file and align comments for all lines involved into description of each member. Or remove the comment, if you arguing that no where else in the file return type lines have comments. 

---
In reply to: [188368103](https://github.com/dotnet/roslyn/pull/26846#discussion_r188368103) [](ancestors = 188368103,188366864,188352605)

Reply:
Please do not mark this thread as resolved, because the concern is not addressed 

---
In reply to: [188373239](https://github.com/dotnet/roslyn/pull/26846#discussion_r188373239) [](ancestors = 188373239,188368103,188366864,188352605)